packages/flutter/lib/src/painting/debug.dart

```diff
@@ -195,3 +195,27 @@ bool debugAssertAllPaintingVarsUnset(String reason, { bool debugDisableShadowsOv
   }());
   return true;
 }
+
+/// The signature of [debugCaptureShaderWarmUpPicture].
+///
+/// Used by tests to run assertions on the [Picture] created by
+/// [ShaderWarmUp.execute]. The return value indicates whether the assertions
+/// pass or not.
+typedef ShaderWarmUpPictureCapture = bool Function(Picture);
```

Comment:
If it's a signature specifically for `debugCaptureShaderWarmUpPicture`, should it be named `DebugShaderWarmUpPictureCapture`?

Reply:
Maybe this would be better as `ShaderWarmUpPictureCallback`.